eo-maven-plugin/src/main/java/org/eolang/maven/CompileMojo.java

```diff
@@ -190,7 +222,7 @@ private void compile(final Path file) {
                 ex
             );
         }
-        Logger.info(this, "%s compiled to %s", file, this.generatedDir);
+        Logger.info(this, "%s compiled to %s with the original compiler", file, this.generatedDir);
```

Comment:
@nlchar I'm not sure this message is accurate. Maybe it's better to say here what was the compiler, using `this.compiler` attribute?

Reply:
@yegor256  Actually, it is correct. This method is used for the original compiler's pipeline. 
Ours logs its own message.
Have a look: https://github.com/HSE-Eolang/transpiler/blob/5fc36687e8a12b07861220324df3495dc261b0e2/src/main/java/io/github/hse_eolang/transpiler/Transpiler.java#L69